lib/hanami/action/application_action.rb

```diff
@@ -51,6 +67,11 @@ def configure_action(action_class)
       end
 
       module InstanceMethods
+        # FIXME: Can I turn these into attr_readers?
```

Comment:
@timriley yes, what was your concern?

Reply:
Mostly a "note to self" that I never got around to adjusting. I'll do this now :)

Reply:
Fixed :)